package.json

```diff
@@ -50,9 +50,9 @@
     "notistack": "1.0.10",
     "opener": "1.5.2",
     "qrcode.react": "1.0.1",
-    "react": ">= 17.0.2",
+    "react": ">=17.0.0 || >=18.0.0",
```

Comment:
Why the `||`? I mean `18` is greater then `17`...

Reply:
yes but `>=` operator does not work for different major version so with `>=17.0.0` it only includes versions starting with `17.x.x`. React version `18.x.x` would not be considered.